src/uniprotkb/components/entry/__tests__/__snapshots__/EntryMain.spec.tsx.snap

```diff
@@ -3027,6 +3054,15 @@ exports[`Entry view should render for non-human entry 1`] = `
       <div
         class="card__content"
       >
+        <h3
```

Comment:
You really have eagle eyes 🦅 
I did notice it, but figured it wouldn't be important, because it's just noticeable when comparing before and after.
This is because we're generating headings at the component level and I couldn't override this one, so I just visually hid its content but it's margin is still there.